src/main/java/com/github/alexcojocaru/mojo/elasticsearch/v2/util/ProcessUtil.java

```diff
@@ -316,6 +328,11 @@ public static boolean isWindowsProcessAlive(InstanceConfiguration config, String
         // and unsets these. And because the scripts would print a warning, we can't rely on the output :(
         result.remove("JAVA_TOOL_OPTIONS");
         result.remove("JAVA_OPTS");
+        // the following environment variables may interfere with the elasticsearch instance.
+        result.remove("ES_HOME");
+        result.remove("ES_JAVA_OPTS");
```

Comment:
If the ES_JAVA_OPTS was set in InstallPluginsStep:28 (through a plugin config param), this line will remove that setting.

Reply:
Missed that bit, sorry. Will fix.

Reply:
Maybe a better approach is to remove the troublesome env vars before setting whichever vars (eg. ES_JAVA_OPTS) were set on the plugin config.